docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,368 @@
+PREDICTION_DURATION_IN_SLOTS=50400
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days
+NODE_OPERATOR_NETWORK_PENETRATION_THRESHOLD_BP=100 # 1% network penetration for a single NO
+```
+
+## OracleReportSanityChecker
+
+```python
+churnValidatorsPerDayLimit = 40000
+oneOffCLBalanceDecreaseBPLimit = 500
+annualBalanceIncreaseBPLimit = 1000
+simulatedShareRateDeviationBPLimit = 50
+maxValidatorExitRequestsPerReport = 500
+maxAccountingExtraDataListItemsCount = 500
+maxNodeOperatorsPerExtraDataItemCount = 100
+requestTimestampMargin = 7680
+maxPositiveTokenRebase = 750000
+```
+
+## Burner
+
+```python
+# proofs:
+# - https://vote.lido.fi/vote/106
+# - https://etherscan.io/address/0xB280E33812c0B09353180e92e27b8AD399B07f26#readContract#F7
+totalNonCoverSharesBurnt = 32145684728326685744
+
+totalCoverSharesBurnt = 0  
+_admin = 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c  
+_treasury = 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c  
+_stETH = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
+```
+
+## DepositSecurityModule
+
+```python
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F10
+maxDepositsPerBlock = 150
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F11
+minDepositBlockDistance = 25
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F14
+pauseIntentValidityPeriodBlocks = 6646
+```
+
+## AccountingOracle
+And its corresponding `HashConsensus`.
+
+```python
+epochsPerFrame = 225  # once per day
+# So, the AccountingOracle expected report time would be 12:00 UTC
+
+fastLaneLengthSlots = 10
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+lido = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
+legacyOracle = 0x442af784A788A5bd6F42A01Ebe9F287a871243fb
+```
+
+## ValidatorsExitBusOracle
+And its corresponding `HashConsensus`.
+
+```python
+epochsPerFrame = 75  # thrice per day
+# So, the ValidatorsExitBusOracle expected report time would be 
+# 4:00 UTC, 12:00 UTC, 20:00 UTC
+
+fastLaneLengthSlots = 10
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+```
+
+## AccountingOracle and ValidatorsExitBusOracle
+
+```python
+secondsPerSlot = 12
+genesisTime = 1606824023
+```
+
+## WithdrawalQueueERC712
+
+```python
+_wstETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0
+_name = "stETH Withdrawal NFT"
+_symbol = "unstETH"
+_tokenURI = "https://wq-api.lido.fi/v1/nft"
+```
+
+## WithdrawalsVault
+
+```python
+_lido = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84 
+_treasury = 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c
+```
+
+## EIP712StETH
+
+```python
+_stETH = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
+```
+
+## StakingRouter
+
+```python
+_depositContract = 0x00000000219ab540356cBB839Cbe05303d7705Fa
+```
+
+## GateSeal
+
+```python
+seal_duration = 518400  # 6 days as seconds (2 x gov time)
+expiry_timestamp = 1714521600  # 2024-05-01 00:00 UTC
+```
+
+## Roles setup
```

Comment:
In general would be great to show the actions and their source (actors) for each role (especially new ones)